src/game/player_actions.c

```diff
@@ -36,7 +36,7 @@ void sub_801561C(void)
         oldPlayerMovestate |= MOVESTATE_80000000;
     }
 
-    for (i = 0; i < ARRAY_COUNT(gUnknown_030056B0); i++) {
+    for (i = 0; i < (s32)ARRAY_COUNT(gUnknown_030056B0); i++) {
```

Comment:
No idea why this is necessary, but alright.
In the future it might make sense to have the cast included in the macro, provided that it doesn't interfere with matching code anywhere.